conda_pack/core.py

```diff
@@ -1002,8 +1002,10 @@ def rewrite_conda_meta(source):
         script_dir = os.path.dirname(__file__)
         new_prefix = os.path.abspath(os.path.dirname(script_dir))
         for path, placeholder, mode in _prefix_records:
-            update_prefix(os.path.join(new_prefix, path), new_prefix,
-                          placeholder, mode=mode)
+            new_path = os.path.join(new_prefix, path)
+            if on_win:
```

Comment:
```suggestion
            if on_win and mode == 'text':
```

Do we need the same construct here too?

Reply:
I tried this, and found some instances that raised the same error, so I believe now that the issue is more general.

Reply:
In that case should the _other_ instance of this `if` clause be adjusted to not have the `mode == 'text'` check?

Reply:
Perhaps, although given the failure is rising in particular from `open(...)`, where `new_prefix` (from this scope) is never used, I would suggest these conversions serve different purposes - I would imagine altering the representation of Windows filepaths to Linux filepaths isn't as well supported in Windows binary applications as in Python. Regardless, I feel more comfortable making the minimal change, and the above change doesn't propagate farther than the current Python process.